Bottom: abf1866b7894e797cfda92f5727e149d39e30c27
Top:    cbba9d1e7bfbfa09e40d3a2e82f18247eecad6e6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:00:34 +0000

Refresh of alternative-more-thorough-step

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index b2539ca..f4688ac 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -21,7 +21,6 @@
 #include "hid/common/contour3d.h"
 #include "hid/common/appearance.h"
 #include "hid/common/face3d.h"
-#include "hid/common/edge3d.h"
 #include "hid/common/object3d.h"
 
 #ifndef WIN32
